pkg/vmprovider/providers/vsphere/vmprovider_vm_test.go

```diff
@@ -247,6 +247,44 @@ func vmTests() {
 				})
 			})
 
+			Context("VM Class spec CPU reservation & limits are non-zero and ConfigSpec specifies CPU reservation", func() {
+				BeforeEach(func() {
+					vmClass.Spec.Policies.Resources.Requests.Cpu = resource.MustParse("2")
+					vmClass.Spec.Policies.Resources.Limits.Cpu = resource.MustParse("2")
+
+					// Specify a CPU reservation via ConfigSpec
+					rsv := int64(6)
+					configSpec = &types.VirtualMachineConfigSpec{
+						Name: "dummy-VM",
```

Comment:
It is set on other `VirtualMachineConfigSpec` in this file but `Name` isn't actually required right?

Reply:
yeah. it's not. it's always overwritten by the VM spec name.